src/styles/components/operative/work-order.scss

```diff
@@ -29,13 +29,19 @@
   }
 }
 
-.truncate {
+.truncate-description {
   display: -webkit-box;
-  -webkit-line-clamp: var(--line-clamp, 3);
   -webkit-box-orient: vertical;
   word-break: var(--word-break, 'none');
   overflow: hidden;
 }
+.truncate-line-3 {
+  -webkit-line-clamp: var(--line-clamp, 3);
+}
+
+.truncate-line-5 {
+  -webkit-line-clamp: var(--line-clamp, 5);
+}
```

Comment:
How about
```
.truncate-line {
  &-3 {
     -webkit-line-clamp: var(--line-clamp, 3);
   }

   &-5 {
     -webkit-line-clamp: var(--line-clamp, 5);
   }
 }
 ```

Reply:
Mm, do you think this version is more readable? I prefer the original. The ampersand trick is neat but here I don't think it adds readability to these tiny utility classes. The number is a bit lost among the other symbols.

Reply:
yes, I agree with Neil 
I had it before with ampersand (not with numbers though), and it seems like having a utility class is more readable